### DISCUSSION TITLE
How to send a comment for a post in the channel?

### DISCUSSION THREAD

**Discussion by satisfyingdeveloper** [telegraf/telegraf#1875] [Q&A]
Somebody recommended using message_thread_id, but it doesn't work

**Comment by MKRhere** [comment]
The bot has to be in the linked discussion group, and it can receive updates from the channel in the group chat. These messages will have `ctx.message.is_automatic_forward`, so you know these are the messages you should reply to. Replies to these messages will become comments on the channel post.
